src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -217,6 +218,7 @@ object Group {
       group is noAppsAndGroupsWithSameName
       (group.id.isRoot is false) or (group.dependencies is noCyclicDependencies(group))
       group is validPorts
+      group is VolumesModule.validGroup
```

Comment:
could you be more specific (w/ respect to both comments)? @aquamatthias recommended that I look at service port validation for inspiration, which I did.